### ISSUE TITLE
Incorrect color of header and footer navigation links

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/wporg-mu-plugins#213]
The navigation links in the header for wordpress.org and make.wordpress.org should be white and instead are displaying as #21759b. This is resulting in a very poor accessibility color contrast rating.

<img width="1244" alt="image" src="https://user-images.githubusercontent.com/1017872/177651531-7fbda2fc-7c16-47cf-91e6-f035d612b923.png">

Color contrast score: https://coolors.co/contrast-checker/0072a9-23282d
![Screen Shot 2022-07-07 at 10 01 10 AM](https://user-images.githubusercontent.com/1017872/177651467-07f56e40-c228-45b3-a1ff-689d0614e838.jpg)

**Comment by iandunn** [comment]
I'm seeing it after a hard refresh. I'm guessing this is another case where something in the latest Gutenberg plugin changed, and either:

1. exposed an instance of us doing something wrong; or
2. broke back-compat

That's been an ongoing problem since we launched the header/footer. 

It looks like the problem is that `wp-block-navigation-item__label` is inheriting `color` from https://s.w.org/wp-content/themes/pub/wporg-main/css/style.css?ver=1653615553 line `690`. It doesn't happen on `/news`, since that site doesn't load the `wporg-main` theme. I'm guessing that before Gutenberg 13.6.0 there was some style that applied the `color: white`, but now that's gone, or is different.

A good quick fix might be just adding something like this to `global-header-footer/postcss/header/menu.pcss`:
```css
.wp-block-group.global-header .global-header__navigation .wp-block-navigation__container .wp-block-navigation-item a {
  color: var(--wp--preset--color--white);
}
```

Once production looks good, we can dig deeper into the problem and look into a more "correct" fix.

**Comment by adamwoodnz** [comment]
Yep I've discovered Gutenberg 13.6.0 removed

```
.wp-block-navigation .wp-block-navigation-item__content {
    color: inherit;
}
```

I'll make the quick fix, thanks

**Comment by adamwoodnz** [comment]
Reopened as this affects the footer links as well
